libraries/ui/src/Nav.tsx

```diff
@@ -14,92 +14,151 @@ export type NavProps = React.PropsWithChildren<{
-            type="button"
-            onClick={() => setIsExpanded(!isExpanded)}
-            className="nav__dropdown-button flex items-center gap-2 hover:text-bluedot-normal"
+    <nav className={clsx('nav fixed z-50 w-full bg-bluedot-canvas container-elevated', className)}>
+      <div className="nav__container w-full max-w-max-width mx-auto px-3 sm:px-6 lg:px-9">
+        <div className="nav__bar flex flex-grow justify-between items-center pl-3 h-[72px] sm:h-[100px]">
+          <a href="/" className="nav__logo-link shrink-0 w-[200px]">
+            {logo ? <img className="nav__logo h-6 mr-auto" src={logo} alt="BlueDot Impact Logo" /> : <p className="nav_logo--placeholder h-8 mr-auto text-xl">BlueDot Impact</p>}
+          </a>
+          <NavLinks
+            onToggleExplore={onToggleExplore}
+            exploreExpanded={exploreExpanded}
+            courses={courses}
+            className="nav__links--desktop hidden lg:flex"
           >
-            Explore
-            <svg
-              className={clsx('size-4 transition-transform', isExpanded ? 'rotate-180' : '')}
-              viewBox="0 0 24 24"
-              fill="none"
-              stroke="currentColor"
-            >
-              <path strokeLinecap="round" strokeLinejoin="round" strokeWidth={2} d="M19 9l-7 7-7-7" />
-            </svg>
-          </button>
-          {children}
-        </div>
-        <div className="nav__cta-container shrink-0 flex justify-end items-center gap-[36px]">
-          <a className="nav__secondary-cta" href="https://course.aisafetyfundamentals.com/alignment?show=login" {...EXTERNAL_LINK_PROPS}>Login</a>
-          <CTALinkOrButton className="nav__primary-cta" url="https://aisafetyfundamentals.com/" {...EXTERNAL_LINK_PROPS}>Get started for free</CTALinkOrButton>
+            {children}
+          </NavLinks>
+          <div className="nav__actions flex">
+            <SearchIcon />
+            <LoginButtons className="nav__login--tablet-desktop gap-6 ml-4 mr-2 hidden sm:flex" />
+            <HamburgerButton open={navExpanded} setOpen={onToggleNav} className="nav__menu--mobile-tablet lg:hidden" />
+          </div>
         </div>
-      </div>
-
-      <div className={clsx(
-        'nav__dropdown overflow-hidden transition-[max-height,opacity] duration-300 flex justify-center items-center pb-7',
-        isExpanded ? 'max-h-96 opacity-100' : 'max-h-0 opacity-0',
-      )}
-      >
-        <div className="nav__dropdown-content flex flex-col gap-[14px] w-max">
-          <h2 className="nav__dropdown-title font-bold text-xl">Our courses</h2>
-          {courses?.map((course) => (
-            <a
-              key={course.href}
-              href={course.href}
-              className="nav__dropdown-link"
+        <div
+          className={clsx(
+            'nav__drawer transition-[max-height,opacity] duration-300 relative overflow-hidden px-3',
+            navExpanded ? 'max-h-[700px] opacity-100' : 'max-h-0 opacity-0',
+          )}
```

Comment:
The [prototype](https://www.figma.com/proto/s4dNR4ELGKPbja6GkHLVJy/Website-Laura's-Working-File?node-id=788-6484&t=yATv1a4ABNtJsXpz-0&scaling=min-zoom&content-scaling=fixed&page-id=744%3A2412&starting-point-node-id=744%3A2413) has spring-like animations for this, the explore button, and the hamburger menu. We could add this with the [motion](https://motion.dev/) library, but I think it looks good as is so I decided to avoid the extra dependency + main thread work